### DISCUSSION TITLE
Autosort Cabinet Contents

### DISCUSSION THREAD

**Discussion by RadCatish** [FakeFishGames/Barotrauma#4837] [Ideas and suggestions]
Please could we have a button which autosorts the contents of large cabinets?

**Comment by mLuby** [comment]
Even simpler would be to just _always_ sort _all_ cabinets when an item is contained or removed. I think it'd take some getting used to though, and I for one prefer a more [flexible approach to cabinet organization](https://github.com/Regalis11/Barotrauma/discussions/4840).
